modules/microshift-custom-ca-cert-cleaning.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * microshift_security_compliance/microshift-custom-ca.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-custom-ca-certificates-cleaning_{context}"]
+= Cleaning up and recreating the custom certificates
+
+To stop the {microshift-short} services, clean up the {microshift-short} certificates and recreate the {microshift-short} certificates, use the following steps.
+
+.Procedure
+
+. Stop the {microshift-short} services and clean up the {microshift-short} certificates by running the following command:
++
+[source,terminal]
+----
+$ sudo microshift-cleanup-data --cert
+----
++
+.Example output
+[source,terminal]
+----
+Stopping MicroShift services
+Removing MicroShift certificates
+MicroShift service was stopped
+Cleanup succeeded
+----
+
+. Restart the {microshift-short} services to recreate the {microshift-short} certificates by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl start microshift
+----
```

Comment:
Should an example output be included here as well?

Reply:
No output is required as it is just starting the service that was stopped. The example output confirms the same.